templates/ts/action/package.json.tpl

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "{{appName}}",
```

Comment:
Should be {{SafeProjectNameLowerCase}}
I suppose all templates' package names are {{SafeProjectNameLowerCase}} now. Do you see any exception?